src/vm/peimagelayout.cpp

```diff
@@ -150,6 +150,10 @@ void PEImageLayout::ApplyBaseRelocations()
     DWORD dwOldProtection = 0;
 
     BOOL bRelocDone = FALSE;
+    BOOL bCrossPageBoundary = FALSE;
+    BYTE * pFlushRegion = NULL;
+    SIZE_T cbFlushRegion = 0;
+    SIZE_T cbPageSize = GetOsPageSize();
```

Comment:
The page size of PE file relocs is always 4k. Do you really need to get the actual OS page size?